src/modules/auth/auth.controller.ts

```diff
@@ -58,4 +60,22 @@ export default class RegistrationController {
     const { id: user_id } = request['user'];
     return this.authService.enable2FA(user_id, password);
   }
+
+  @skipAuth()
+  @Post('signin-token')
+  @ApiOperation({ summary: 'Request Signin Token' })
+  @ApiResponse({ status: 200, description: 'Sign-in token sent to email', type: RequestSigninTokenDto })
+  @ApiResponse({ status: 400, description: 'Bad request' })
+  public async signInToken(@Body() body: RequestSigninTokenDto, @Res() response: Response) {
```

Comment:
You don't seem to need the response decorator here

```suggestion
public async signInToken(@Body() body: RequestSigninTokenDto) {
```

Reply:
Fixed